Areas.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require ("vendor/autoload.php");
+
+use Shape\Circle;
+use Shape\Rectangle;
+use Shape\Square;
+use Shape\RightAngleTriangle;
+use Shape\EquilateralTriangle;
+use Shape\Triangle;
+use Shape\ShapeAreaCalculator;
+
+$name = $_POST["name"];
+$length = $_POST["length"];
+$width = $_POST["width"];
+$sqLength = $_POST["lengthsq"];
+$radius = $_POST["radius"];
+$eqLength = $_POST["eqLength"];
+$rLengthA = $_POST["rLengthA"];
+$rLengthB = $_POST["rLengthB"];
+$lengthA = $_POST["lengthA"];
+$lengthB = $_POST["lengthB"];
+$lengthC = $_POST["lengthC"];
+
+if (isset($_POST["rectangle"])){
+    $shape = new Rectangle($length, $width, $name);
+} elseif (isset($_POST["square"])){
+    $shape = new Square($sqLength, $name);
+} elseif (isset($_POST["circle"])){
+    $shape = new Circle($radius, $name);
+} elseif (isset($_POST["eqTriangle"])){
+    $shape = new EquilateralTriangle($eqLength, $name);
+} elseif (isset($_POST["rTriangle"])){
+    $shape = new RightAngleTriangle($rLengthA, $rLengthB, $name);
+} elseif (isset($_POST["triangle"])){
+    $shape = new Triangle($lengthA, $lengthB, $lengthC, $name);
+}
```

Comment:
Let's do something more advanced with this block of code. I hope you'll like the end result.

Investigate what are enumberators. In PHP they were added in the last version, 8.1 if I'm not mistaken. Refactor your code to use them here (you have these types of shapes).

---

Let's learn some design pattern. Here you're creating instances of class based on some criteria (type of shape). There are various creational design patterns which are meant to help with this problem.
Let's explore them, great resource is this one: https://designpatternsphp.readthedocs.io/en/latest/README.html
After exploration, let's pick one of them on your choice and refactor shape creation code accordingly.
Please do not hesitate to reach out to me on this topic, I'll be happy to help!